src/platform/packages/shared/kbn-scout/src/playwright/config/create_config.ts

```diff
@@ -27,9 +27,31 @@ export function createPlaywrightConfig(options: ScoutPlaywrightOptions): Playwri
     process.env.TEST_RUN_ID = runId;
   }
 
+  const scoutProjects: PlaywrightTestConfig<ScoutTestOptions>['projects'] = [
+    {
+      name: 'local',
+      use: { ...devices['Desktop Chrome'], configName: 'local' },
```

Comment:
each project will override default `configName` to pass source of servers configuration ('local.json', 'cloud-mki', 'cloud-ech', etc)